src/tools/tidy/src/unit_tests.rs

```diff
@@ -20,10 +20,10 @@ pub fn check(root_path: &Path, bad: &mut bool) {
     let mut skip = |path: &Path| {
         let file_name = path.file_name().unwrap_or_default();
         if path.is_dir() {
-            super::filter_dirs(path) ||
-            path.ends_with("src/test") ||
-            path.ends_with("src/doc") ||
-            (file_name == "tests" || file_name == "benches") && !is_core(path)
+            super::filter_dirs(path)
+                || path.ends_with("src/test")
+                || path.ends_with("src/doc")
+                || (file_name == "tests" || file_name == "benches") && !is_core(path)
```

Comment:
I am not sure why this commit is separate, it seems like it should either be dropped from this PR or moved into commit(s) editing this file in this PR?

Reply:
Dropping will make fmt error. Merged as you feel it better.